src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -412,6 +413,10 @@ export class RoslynLanguageServer {
             args.push('--logLevel', logLevel);
         }
 
+        if (this.additionalExtensionPaths.length > 0) {
+            args.push('--additionalExtensions', `"${this.additionalExtensionPaths.join(',')}"`);
```

Comment:
Why --additionalExtensions? I'd expect the existing --extension flag already works, right? If these have to be different for some reason then we should figure out better naming than "extension" and "additionalExtension".

Reply:
I have no idea what existing flags exist :slightly_smiling_face:.